packages/button/docs/IconButton.stories.tsx

```diff
@@ -18,7 +18,7 @@ export default {
     docs: {
       description: {
         component:
-          'import { IconButton } from "@kaizen/button". This IconButton supersedes "@kaizen/draft-button".',
+          'import { IconButton } from "@kaizen/button". This IconButton supersedes "@kaizen/button".',
```

Comment:
Yeah we definitely don't want to change this one 😂

Reply:
omg yep, that would not have be smart, nice catch🧤